medyczna_placowka/settings_local.py

```diff
@@ -5,10 +5,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': config('DB_NAME'),
-        'USER': config('DB_USER'),
-        'PASSWORD': config('DB_PASSWORD'),
-        'HOST': config('DB_HOST'),
-        'PORT': config('DB_PORT'),
+        'NAME': config('DB_NAME', default='DEFAULTDBNAME'),
+        'USER': config('DB_USER', default='DEFAULTUSERNAME'),
+        'PASSWORD': config('DB_PASSWORD', default='DEFAULTPASSWORD'),
+        'HOST': config('DB_HOST', default='DEFAULTHOST'),
+        'PORT': config('DB_PORT', default='1111'),
```

Comment:
Po co te defaulty? DEFAULTDBNAME brzmi jak zmienna środowiskowa więc jeśli ktoś nie określi DB_NAME to jaka jest szansa że określi DEFAULTDBNAME?

Reply:
Usunięte, defaultowe ustawienia, które mają służyć testom na GH wrzucone do workflow.